package org.telegram.ui.Components;

import android.graphics.Canvas;
public final class c41 extends wh.p {
    @Override
    public final void onDraw(Canvas canvas) {
        super.onDraw(canvas);
        canvas.save();
        canvas.translate(getPaddingLeft(), getPaddingTop());
        aw0.a(canvas, getLayout());
        canvas.restore();
    }
}
